templates/antora/template.js

```diff
@@ -0,0 +1,324 @@
+/* jshint latedef:true */
+
+var fs = require('fs');
+var Handlebars = require('handlebars');
+var path = require('path');
+var ZipWriter = require('moxie-zip').ZipWriter;
+var YAML = require('js-yaml')
+var BASE_PATH = process.env.BASE_PATH || '/_data/antora';
+
+var AntoraTemplate = require('./antora.converter.js');
+// correlates to tinymce-docs antora path
+var AntoraNavBaseDir = 'apis/'; 
+
+var namespaceDescriptions = {
+  'tinymce': 'Global APIs for working with the editor.',
+  'tinymce.dom': 'APIs for working with the DOM from within the editor.',
+  'tinymce.editor.ui': 'APIs for registering User Interface components.',
+  'tinymce.geom': 'Various rectangle APIs.',
+  'tinymce.html': 'APIs for working with HTML within the editor.',
+  'tinymce.util': 'Browser related APIs.'
+};
+
+/**
+ * [function description]
+ * @param  {[type]} root   [description]
+ * @param  {[type]} toPath [description]
+ * @return {[type]}        [description]
+ */
+exports.template = function (root, toPath) {
+  var archive = new ZipWriter();
+  var rootTemplate = compileTemplate('root.handlebars');
+  var namespaceTemplate = compileTemplate('namespace.handlebars');
+  var template = compileTemplate('member.handlebars');
+
+  // bind new archive to function
+  addPageToArchive = addPageToArchive.bind(archive); // jshint ignore:line
+
+  // sort types alphabetically
+  var sortedTypes = root.getTypes().sort(function (a, b) {
+    if (a.fullName < b.fullName) return -1;
+    if (a.fullName > b.fullName) return 1;
+    return 0;
+  });
+
+  var YMLNav = getNavFile(sortedTypes);
+  var AdocNav = YMLNavToAdoc (YMLNav);
+
+  addPageToArchive({
+    filename: '_data/nav.yml',
+    content:  YAML.dump(YMLNav)
+
+  });
+
+  addPageToArchive({
+    filename: '_data/moxiedoc_nav.adoc',
+    content: AdocNav
+  });
+
+  // create all json and adoc for each item
+  var pages = sortedTypes.map(getMemberPages.bind(null, root, template))
+  var convertedPages = AntoraTemplate.convert(pages);
```

Comment:
Take the original Pages data structure [{Json data} + {html page templates}] only convert the html handlebar template into asciidoc, convertedPages should contain everything in asciidoc with the accompanying JSON data structure unchanged